### PR TITLE
Update strings.po

### PR DESCRIPTION
**Pull request by kadireksi**
a few new translations added(170907_1531)

### CODE REVIEW

File: language/Turkish/strings.po [hunk 1]
```diff
@@ -1700,7 +1700,7 @@ msgstr ""
 
 #: src/frontend/mame/ui/selmenu.cpp:358
 msgid "BIOS selection:"
-msgstr ""
+msgstr "BIOS seçimi"
```
Comment (cuavas) [inline language/Turkish/strings.po]: Was removing the colon here in the translation intentional?  (It's an honest question, I don't know how punctuation works in Turkish.)

File: language/Turkish/strings.po [hunk 2]
```diff
@@ -1739,31 +1739,31 @@ msgstr ""
...
 
 #: src/frontend/mame/ui/selmenu.cpp:646
 msgid "Graphics: OK, "
-msgstr ""
+msgstr "Grafik: Tamam"
 
 #: src/frontend/mame/ui/selmenu.cpp:649
 msgid "Sound: None"
-msgstr ""
+msgstr "Ses: Yok"
```
Comment (cuavas) [inline language/Turkish/strings.po]: Here "Sound: None" is translated as "Ses: Yok", however previously "None" is translated as "Hiçbiri".  In both cases it's used in the same context (one is used in the default UI, the other with `-ui simple`).  Is there a reason the translations need to be different?

File: language/Turkish/strings.po [hunk 3]
```diff
@@ -2086,7 +2090,7 @@ msgstr ""
 
 #: src/frontend/mame/ui/miscmenu.cpp:630
 msgid "Name:             Description:\n"
-msgstr ""
+msgstr "Ad:             Açıklama:\n"
```
Comment (cuavas) [inline language/Turkish/strings.po]: This is used as a heading in a list file where the first column is padded to a fixed width with spaces.  Could you please add spaces so the column width matches in the translation?  It needs two more spaces.

File: language/Turkish/strings.po [hunk 4]
```diff
@@ -2423,24 +2427,24 @@ msgstr ""
...
 
 #: src/frontend/mame/ui/submenu.cpp:109
 msgid "Window Mode"
-msgstr ""
+msgstr "Pencere Kipi"
 
 #: src/frontend/mame/ui/submenu.cpp:110
-msgid "Enforce Aspect Ratio"
-msgstr ""
+  msgid "Enforce Aspect Ratio"
```
Comment (cuavas) [inline language/Turkish/strings.po]: There seems to be a stray character added to the beginning of this line.  Could you please correct it?

**Comment by kadireksi** [comment]
Hi,
sorry for answering late, real life issues(married with 2 children and 2
different jobs).
I'll correct typo and copy&paste errors in a few days and add new
translations, hopefuly.

Best regards

**Comment by cuavas** [comment]
Thanks for getting back to us, there's no problem with the delay.  Real life is more important than MAME.

File: language/Turkish/strings.po (hunk 1 above)
Comment (kadireksi) [reply language/Turkish/strings.po]: added the colon

File: language/Turkish/strings.po (hunk 2 above)
Comment (kadireksi) [reply language/Turkish/strings.po]: i guess this is used for info and not for choice and means that driver/machine/system has no sound.
"Hiçbiri" is used when choosing something from multiple things and represent 'none of them'.
i hope i understood right and i told it right

File: language/Turkish/strings.po (hunk 3 above)
Comment (kadireksi) [reply language/Turkish/strings.po]: added two more spaces

File: language/Turkish/strings.po (hunk 4 above)
Comment (kadireksi) [reply language/Turkish/strings.po]: deleted the stray character

**Comment by cuavas** [comment]
Thanks very much.

**Pull request merged**